gen3testing.braincommons.org/manifests/fence/fence-config-public.yaml

```diff
@@ -1,37 +0,0 @@
-# Which Identity Provider fence will/can use
-#
```

Comment:
did you erase this on purpose?

Reply:
I replaced the `manifests` folder with `manifests/modsec` folder here! I will add back the `fence-config-public.yaml` back to the manifests folder